src/common/components/lists/ListDefault.js

```diff
@@ -2,48 +2,43 @@ import PropTypes from "prop-types";
 import React from "react";
 import styled from "styled-components";
 
-import right from "../../../assets/icon-right.png";
-
 const StyledListDefault = styled.div`
   width: 100%;
   height: 44px;
   display: flex;
-  justify-content: space-between;
+  justify-content: flex-start;
   align-items: center;
   background-color: black;
   .label {
```

Comment:
저는 클래스로 스타일링을 하게되면 구분을 위해서 개행을 해주는 편인데요. 이부분은 합의가 필요해 보입니다. 
```suggestion

  .label {
```

Reply:
저도 이부분에 있어서는 개행 해주는 편인데, 최대한 진호님이 처음 작성해주신 코드를 건들이지 않는 방향으로 구현했습니다. 해당 부분은 합의가 필요할것 같네요.